rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -3696,7 +3695,7 @@ fn test_install_when_updating_memory_allocation_via_canister_settings() {
 
         let sender = canister_test_id(100).get();
         let settings = CanisterSettingsBuilder::new()
-            .with_memory_allocation(MemoryAllocation::try_from(NumBytes::from(2)).unwrap())
+            .with_memory_allocation(MemoryAllocation::try_from(NumBytes::from(200)).unwrap())
```

Comment:
It is necessary for the test to pass since the canister uses more memory than 2B.